lib/workers/repository/process/lookup/index.ts

```diff
@@ -71,6 +71,13 @@ export async function lookupUpdates(
     if (dependency.deprecationMessage) {
       logger.debug({ dependency: depName }, 'Found deprecationMessage');
       res.deprecationMessage = dependency.deprecationMessage;
+      if (dependency.replacementName && dependency.replacementVersion) {
+        res.updates.push({
```

Comment:
Missing coverage 😏

Reply:
Thanks! This actually helped me catch an implementation bug. We discussed above that not only packages with a `deprecationMessage` should be allowed to have replacements, but, for example you might want to replace public Docker images with internal versions. I've moved this if statement outside if the `if (dependency.deprecationMessage)`

These are the mistakes that happen on such a long-lived branch!